src/gui/TimeLineWidget.cpp

```diff
@@ -274,17 +273,6 @@ void TimeLineWidget::paintEvent( QPaintEvent * )
 			p.drawText( cx + 5, ((height() - fontHeight) / 2) + fontAscent, s );
 		}
 	}
-
-	// Draw the markers
-	p.setPen( QColor( 0, 0, 0 ) );
-	p.setOpacity( loopPointsEnabled() ? 0.9 : 0.2 );
-	int const loopMarkerHeight = ( this->height() - s_loopPointBeginPixmap->height() ) / 2;
-	p.drawPixmap( markerX( loopBegin() ) + 2, loopMarkerHeight, *s_loopPointBeginPixmap );
-	p.drawPixmap( markerX( loopEnd() ) + 2, loopMarkerHeight, *s_loopPointEndPixmap );
-
-	p.setOpacity( 0.6 );
-	p.drawPixmap( m_posMarkerX, height() - s_posMarkerPixmap->height(),
```

Comment:
@michaelgregorius you accidentaly removed the playhead marker pixmap here:
![screenshot from 2016-05-08 23 08 45](https://cloud.githubusercontent.com/assets/6282045/15100439/dfcc7966-1571-11e6-8d2d-9bd0f89e1a1f.png)